dash-pipeline/bmv2/dash_metadata.p4

```diff
@@ -28,11 +28,18 @@ struct conntrack_data_t {
     bool allow_out;
 }
 
+enum bit<16> dash_tunnel_dscp_mode_t {
+    INVALID = 0,
+    PRESERVE_MODEL = 1,
```

Comment:
should we make one of preserve_model or explicit set as default instead of invalid?

Reply:
also I guess you mean `PRESERVE_MODE` : D

Reply:
Actually, please ignore my comment, I think what you are doing is right.